resources/ts/Components/Footer.tsx

```diff
@@ -1,9 +1,109 @@
 import React from "react";
+import { PubliqLogo } from "./logos/PubliqLogo";
+import { VlaanderenLogo } from "./logos/VlaanderenLogo";
+import { Heading } from "./Heading";
+import { ButtonLink } from "./ButtonLink";
+import { Link } from "@inertiajs/react";
+import { useTranslation } from "react-i18next";
 
 export default function Footer() {
+  const { t } = useTranslation();
   return (
-    <footer className="p-3 text-center">
-      <p>Copyright 2022 publiq</p>
+    <footer
+      className="bg-publiq-blue text-white w-full flex justify-between px-7 py-10 max-md:flex-col gap-7"
+      data-component-name="footer"
+    >
+      <div className="flex-col">
+        <PubliqLogo color="#FFFFFF" width={140} height={114} />
+        <div className="flex flex-col gap-7">
+          <div className="flex flex-col gap-1">
+            <Heading level={3} className="font-medium">
+              {t("global.publiq")}
+            </Heading>
+            <address className="font-light not-italic tracking-wide">
+              {t("footer.address.part1")}
+              <br />
+              {t("footer.address.part2")}
+            </address>
+          </div>
+          <div className="flex flex-col gap-1">
+            <div className="flex gap-3">
+              <span>{t("global.btw")}</span>
+              <span className="font-extralight">BE 0475 250 609</span>
+            </div>
+            <div className="flex gap-3">
+              <span>{t("global.IBAN")}</span>
```

Comment:
```suggestion
              <span>{t("global.iban")}</span>
```

This one can also be in the same casing like btw